README.md

```diff
@@ -78,7 +78,7 @@ sudo mv inav-configurator.desktop /usr/share/applications/
 
 #### Notes
 
-On some Linux distros, you may be missing `libatomic`, an `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
+On some Linux distros, you may be missing `libatomic`or `NW.JS` (especially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
```

Comment:
I think is should be **and/or** because it could be both, one, or none. Also, **dependency** should be **dependencies** because it should be plural.